linux_os/guide/services/nfs_and_rpc/package_nfs-utils_removed/rule.yml

```diff
@@ -21,6 +21,9 @@ identifiers:
     cce@rhel7: 82933-3
     cce@rhel8: 82932-5
 
+references:
+    srg: SRG-OS-000095-GPOS-00049
```

Comment:
Is it going to be dropped from the OSPP profile?  Since the STIG is tied to OSPP should there be at a minimum a 1:1 mapping of OSPP rules to SRG?  I can move this to SRG-OS-000480-GPOS-00227 if that works.

Reply:
@stevegrubb can you help why `nfs-utils` are removed by OSPP profile?

Reply:
NFS is not part of the TOE. It opens rpc ports which we do not want exposed. Therefore all parts of it needs to go.

Reply:
Created https://github.com/ComplianceAsCode/content/issues/5014 to track if DoD STIG should truly disable NFS utils.